CMakeLists.txt.in

```diff
@@ -0,0 +1,215 @@
+cmake_minimum_required(VERSION 3.5)
```

Comment:
should this match ours ? to 3.9? why have 2 separate versions. maybe we use a variable like @CMAKE_VERSION@ or smth

Reply:
they should be same, fixed.